### ISSUE TITLE
Final class constants (aka. `final public const`) should stay final on PHP >= 8.1

### ISSUE THREAD

**Issue by jdeniau** [prettier/plugin-php#2183]
final public const should not remove the `final` keyword as it is valid since PHP 8.1

**@prettier/plugin-php v0.19.4**
[Playground link](https://loilo.github.io/prettier-php-playground/#N4IgDgTgpgLjCWUIHkwIPYDsDOIBcok8mMA6vACYwAW+AHAAwA0IMAhgEblW14AsLAK7YoAFU648AMzYAbES2zEA5rKgBFQehhR8M+VBZhqYAGpIlWfCDoA6AIwgWMCG3iyVAYXQBbH2wAFAAkA-BdBQxAOVwBjKABlGABPNWswJABaGNwWaABHQXhoANdlfz05BRBiEQgYErYytgqDAF8WKAp4GHQUNHgsSVA2bBgWkXbqzDBBMbwQAB4AfmMwAB1MDZjZEewAAgAxdHQ94A29valiOT2Zjg8YvZjBmEPkZD2AXj21kAP334Abg2rRArSAA)

**Input:**
```php
<?php

class Foo {
  final public const FOO = "FOO";
}
```

**Output:**
```php
<?php

class Foo
{
    public const FOO = "FOO";
}

```

See [release note on PHP 8.1](https://www.php.net/releases/8.1/en.php#final_class_constants)

Thank you

**Comment by cseufert** [comment]
This also requires upstream fix in php-parser to be merged
https://github.com/glayzzle/php-parser/pull/1110

**Comment by czosel** [comment]
Fixed in `v0.19.6` :tada:
